src/UserSpaceInstrumentation/AccessTraceesMemory.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "AccessTraceesMemory.h"
+
+#include <absl/strings/numbers.h>
+#include <absl/strings/str_format.h>
+#include <absl/strings/str_split.h>
+#include <sys/ptrace.h>
+
+#include <cerrno>
+#include <cstring>
+#include <string>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ReadFileToString.h"
+#include "OrbitBase/SafeStrerror.h"
+
+namespace orbit_user_space_instrumentation {
+
+using orbit_base::ReadFileToString;
+
+[[nodiscard]] ErrorMessageOr<void> ReadTraceesMemory(pid_t pid, uint64_t address_start,
+                                                     uint64_t length, std::vector<uint8_t>* bytes) {
+  // Round up length to next multiple of eight.
+  length = ((length + 7) / 8) * 8;
```

Comment:
I would define a constant for the 8, something like `kPeekdataReadSize` or similar.

Reply:
`kEight` maybe? :) I don't think that helps.

Reply:
Well no, but that's because it repeats the content. 
But it helps if it reflects the reason for the content, for example why you round to the next multiple of 8. If I get it correctly this is because `PTRACE_PEEKDATA` reads "a **word** at the address addr in the tracee's memory". That's why I suggested `kPeekdataReadSize`.